app/core/config.py

```
@@ -187,44 +88,46 @@ def load_system_prompts() -> dict[str, str]:
+        kwargs = {
+            "model_name": model_name
             }
-            client_cache['databricks'] = ChatProxyClient(base="databricks",**kwargs)
-            logger.info("Databricks client initialized successfully")
-        else:
-            raise ValueError("Invalid Databricks configuration")
+        client = ChatProxyClient(base="ollama", **kwargs)
+    elif model_type == "databricks":
+        model_name = os.environ.get("DATABRICKS_DEFAULT_MODEL")
+        if not model_name:
+            raise ValueError("OLLAMA_DEFAULT_MODEL environment variable is not set")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)